docs/FAQ/how-to-deploy-bold-bi-in-eks-using-application-load-balancer.md

```diff
@@ -0,0 +1,220 @@
+          ---
+        </details>
+           
+    * If you need to use the **Bing Map** widget feature, enter the value for `widget_bing_map_enable` environment variable as `true` and the API key value for `widget_bing_map_api_key`.
+       
+       ![Enable-Bingmap](../images/deploy/eks/bingmap_enable.png)
+
+11. If you have a DNS to map with the application, proceed to the next steps, otherwise, skip to **Step 13**. 
+
+12. Download the `ingress_alb.yaml` file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/main/deploy/deploy/ingress_alb.yaml) Uncomment the host value and replace your DNS hostname with `example.com` in ingress_alb.yaml file in line **18**.
+
+      ![DNS](../images/faq/ingress_alb_host.png)
+
+13. If you have the SSL certificate for your DNS and need to configure the site with it, follow these steps of section A or B, otherwise, skip to **Step 13**.
+
+    **A. SSL configuration using TLS secret:**
+	* Run the following command to create a TLS secret with your own SSL certificate.
+
+	```sh
+	kubectl create secret tls bold-tls -n bold-services --key <key-path> --cert <certificate-path>
+	```
+        
+	* Now, uncomment the `tls` section and replace your DNS hostname with `example.com` in ingress spec and save the file.
+
+	![ingress DNS](../images/faq/tls_ingress_alb.png)
+	
+    **B. SSL configuration using ACM ARN:**
+       * Create an ACM Certificate for your domain and verify ownership. Note down the Amazon Resource Name (ARN) of the Certificate you just created. You can find the ARN in the ACM Management Console, as shown below.
+       
+       ![ACM ARN](../images/faq/AWS_ACM_ARN.png)
+       
+       * Now, uncommand the below lines and replace your Arn with `Amazon Resource Name (ARN) of the Certificate` in ingress annotation and save the file.
+
+	![ACM ARN](../images/faq/ingress_ARN.png)	
+
+14. Apply the `ingress_alb.yaml` by running the below command.
+
+	```sh
+	kubectl apply -f ingress_alb.yaml
+	```
+15. Now, run the following command to get the ingress address.
+
+	```sh
+	kubectl get ingress -n bold-services
+	```
+	Repeat the above command till you get the value in ADDRESS tab.
+	![Ingress Address](../images/faq/ingress_alb_address.png) 
+	
+16. Replace your DNS or Ingress address in `<application_base_url>` place of the **deploy_eks_alb.yaml** file with http or https protocal.
+
+    Ex:  `http://example.com`, `https://example.com`, `http://<ingress_address>`
+    
+    ![App_Base_Url](../images/faq/alb_app_base_url.png) 
+    
+17. Read the optional client library license agreement from the following link.
+
+    [Consent to deploy client libraries](../docs/consent-to-deploy-client-libraries.md)
+    
+18. By default, all client libraries for Bold BI will be installed in Kubernetes. However, you can overwrite them by specifying the required libraries as a comma-separated list in the environment variable noted from the above link.
```

Comment:
18. By default, all client libraries for Bold BI will be installed in Kubernetes. However, overwrite them by specifying the required libraries as a comma-separated list in the environment variable noted from the above link.

Reply:
concern addressed